core/modules/widgets/navigator.js

```diff
@@ -641,7 +641,7 @@ NavigatorWidget.prototype.handleManageACLTiddlersEvent = function() {
 	var paths = pathname.split("/");
 	var recipeName = paths[paths.length - 1];
 	var bagName = document.querySelector("h1.tc-site-title").innerHTML;
-	window.location.href = "/admin/manage-acl/"+recipeName+"/"+bagName
+	window.location.href = "/admin/acl/"+recipeName+"/"+bagName
```

Comment:
I think the API paths should not be so specific. eg: `/admin/` is very specific. 

I would suggest `/api/v1/acl/*` as base path. Which would make it consistent and it implies "admin" so it can be skipped.

Reply:
Bear in mind that our objective with this PR is to find the shortest path to enable us to merge it. These considerations are something that we can come back to.